src/content/releases/previous_releases.md

```diff
@@ -0,0 +1,17 @@
++++
+date = 2020-07-09T08:53:23-04:00
+title = "Previous Releases"
++++
+
+The following releases are out of date and should only be used if your
+environment specifically requires them. Please consider using the newest
+release in the series instead.
+
+| Series | Release | Release Date |
+| ------ | ------- | ------------ |
+| OVN 20.03 | [OVN 20.03.0](https://github.com/ovn-org/ovn/releases/tag/v20.03.0) | 02 Mar 2020 |
+| OVN 20.06 | [OVN 20.06.0](https://github.com/ovn-org/ovn/releases/tag/v20.06.0) | 09 Jun 2020 |
```

Comment:
NIT: prob does not matter, but on the other page, the releases are listed in the inverse order.
I do not mind either way -- and this should be automated anyways -- but maybe make it consistent on
both pages?

Reply:
Done!